pkg/executables/clusterctl.go

```diff
@@ -26,11 +30,14 @@ const (
 	etcdadmBootstrapProviderName  = "etcdadm-bootstrap"
 	etcdadmControllerProviderName = "etcdadm-controller"
 	kubeadmBootstrapProviderName  = "kubeadm"
+	clusterctlMoveTimeout         = 30 * time.Minute // Arbitrarily established.  Equal to kubectl wait default timeouts.
```

Comment:
Did we check if `clusterctl` has some default timeouts on their end?

Reply:
clusterctl establishes timeouts for the individual API operations it performs, but I don't see any overarching timeout for the entire move operation, which this timeout is the equivalent of.